code-samples/grant/node/update-grants/index.js

```diff
@@ -0,0 +1,103 @@
+import * as dotenv from "dotenv";
+import Nylas from "nylas";
+import inquirer from "inquirer";
+
+dotenv.config();
+
+const nylas = new Nylas({
+  apiKey: process.env.API_KEY,
+});
+
+const grants = await nylas.auth.grants.list();
+
+async function main() {
+  if (grants.data.length === 0) {
+    console.log("No grants found!");
+    return;
+  }
+
+  console.clear();
+  try {
+    const answer = await inquirer.prompt([
+      {
+        type: "list",
+        name: "grant",
+        message: "Please select an Gmail account to update:",
+        choices: grants.data.map(({ email, id }) => ({
+          name: email,
+
+          value: {
+            id,
+            email,
+          },
+        })),
+      },
+      {
+        type: "checkbox",
+        name: "scopes",
+        message: "Scopes:",
+        choices: [
+          {
+            name: "Gmail ReadOnly",
+            value: "gmail.readonly",
+          },
+          {
+            name: "Gmail Send",
+            value: "gmail.send",
+          },
+          {
+            name: "Gmail Modify",
+            value: "gmail.modify",
+          },
+          {
+            name: "Calendar",
+            value: "calendar",
+          },
+          {
+            name: "Contacts",
+            value: "contacts",
```

Comment:
@amit-ksh - Does this take into account if the email account has scopes already active?

Reply:
I don't understand, can you explain what do you mean?

Reply:
So as an example, if the selected email has `gmail.modify` already active, do we show this option as active in the following list: 
```

 Gmail ReadOnly
 ◯ Gmail Send
 ◯ Gmail Modify
 ◯ Calendar
 ◯ Contacts

```

Reply:
No, it will not the active status, I can add this you want.

Reply:
Done!